manga_auto_translator/segmentation_Unet/predict.py

```diff
@@ -0,0 +1,89 @@
+# USAGE
+# python predict.py
+# import the necessary packages
+import copy
+
+from pyimagesearch import config
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import cv2
+import os
+
+def prepare_plot(origImage, origMask, predMask,name):
```

Comment:
It may be more convenient to add this function in a specific module in his own .py file => For the maintenance purpose.